### ISSUE TITLE
The Debug button and dropdown is not reachable via keyboard

### ISSUE THREAD

**Issue by ramya-rao-a** [Microsoft/vscode#16720]
- VSCode Version: 1.8 insiders dated 12/6
- OS Version: All

Steps to Reproduce:

1. Tab through and try to reach the debug button

**Expected**: After the "Extensions" icon in the activity bar the next tab stop should be the debug button. The next one after that should go to the drop down and user should be able to interact with the drop down.
**Actual**: After the "Extensions" icon in the activity bar, the next tab stop goes to an unknown land, the next one goes to the gear icon.

**Comment by isidorn** [comment]
Fixed in latest
